cmake/podioConfig.cmake.in

```diff
@@ -42,3 +42,7 @@ if(NOT TARGET podio::podio)
 endif()
 
 check_required_components(podio)
+
+get_property(TEST_PODIO_LIBRARY TARGET podio::podio PROPERTY LOCATION)
```

Comment:
It is better to add
```
include(FindPackageHandleStandardArgs)
```
There are other packages that include this before, but no guarantee for that.

Reply:
Good point, done.